src/molecules/HeaderDiscount/header_discount.module.scss

```diff
@@ -12,18 +10,22 @@
 
 .amount {
   @include typography(1);
-
-  font-weight: bold;
+  @include typography-semibold;
 }
 
-.sup {
-  @include typography-5(false);
+.prev-amount,
+.unit,
+.curr {
+  @include typography-regular;
+  @include typography-semibold;
 
-  vertical-align: super;
+  font-size: 60%;
+  line-height: inherit;
 }
 
-.sm {
-  @include typography-5(false);
+.curr {
+  position: relative;
+  vertical-align: 10%;
```

Comment:
@patrickkim This seems like all repeated styles? Is there a way to give them a single source of truth?